Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -157,7 +176,14 @@ def livecheck_formulae(formulae_to_check, args)
         puts "No newer upstream versions."
       end
 
-      puts JSON.generate(formulae_checked.compact) if args.json?
+      return unless args.json?
+
+      if progress
+        progress.finish
+        print "#{Tty.up}#{Tty.erase_line}" * 2
```

Comment:
```suggestion
        $stderr.print "#{Tty.up}#{Tty.erase_line}" * 2
```

This should print to $stderr now, correct?

Reply:
It works even otherwise which is why I didn't change it, but I'll make the change now.

Reply:
Ah, it seemed to work without it normally but I realise I didn't check the redirection part. I've made the change now.